test/vanilla/Expected/AcceptanceTests/Constants/constants/models/_models.py

```diff
@@ -164,47 +164,33 @@ def __init__(self, **kwargs):
 class NoModelAsStringNoRequiredOneValueDefault(msrest.serialization.Model):
     """NoModelAsStringNoRequiredOneValueDefault.
 
-    Variables are only populated by the server, and will be ignored when sending a request.
-
-    :ivar parameter:  Default value: "value1".
-    :vartype parameter: str
+    :param parameter:
```

Comment:
[swagger](https://github.com/Azure/autorest.testserver/blob/main/swagger/constants.json#L100). Since this is an optional constant with an `x-ms-client-default`, we default to the client default.

The change in description shows me our descriptions can be improved, filed a separate issue for it [here](https://github.com/Azure/autorest.python/issues/953).